.travis.yml

```diff
@@ -112,7 +112,7 @@ matrix:
       sudo: required
       addons:
       before_install:
-      - docker pull balabit/syslog-ng-ubuntu-bionic
+      - docker pull syslog-ng/syslog-ng-ubuntu-bionic
```

Comment:
Unfortunately we do not have a syslog-ng namespace under DockerHub

Reply:
Can we register that?